week02/valid_parentheses.cpp

```diff
@@ -0,0 +1,31 @@
+// TC: O(N)
+// MC: O(N)
+class Solution
+{
+public:
+    bool isValid(char opn_bracket, char closing_bracket)
```

Comment:
You have two different functions doing completely two different work. And still you named them EXACTLY same. :)